BackofficeBundle/Form/Type/Component/ContentSearchType.php

```diff
@@ -4,29 +4,44 @@
 
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
+use OpenOrchestra\Transformer\LuceneToBddTransformer;
 
 /**
  * Class ContentSearchType
  */
 class ContentSearchType extends AbstractType
 {
+    protected $transformer;
+
+    /**
+     * @param string $transformerClass
+     */
+    public function __construct($transformerClass)
```

Comment:
the injection of transformer is weird, how an integrator wich use an onether database than mongo knows to make a transformer which take the field in constructor ?